### ISSUE TITLE
Latest 0.134.0 broke M3 button to change modes and custom are gone.

### ISSUE THREAD

**Issue by faslanetech** [seerge/g-helper#1655]
Just an FYI: The latest update (0.134.0) broke my M3 button from the ability to toggle through the various modes, silent, Turbo, balanced and my custom modes as well as deleted the custom ones I'd saved entirely. Asus M16 2021 model Win 11 Pro 32H2 25997.1010 Maybe this is normal when updating to have to recreate the button mapping and custom modes. I was able to create new customs and reassign the M3 key to "mode" but it did default everything upon updating it.  :-)   Just an FYI not so much as a bug per se. Thank you for such a great little utility!!

Fas

**Comment by seerge** [comment]
@faslanetech hello,

I assume you had a custom M3 binding (as by default it would toggle mic).

What probably happened is that your config file got corrupted by some reason (mb cause app was writing something there the moment it was closed or so), so app has recreated it from scratch afterwards. That would reset all your settings, including M3 action and modes.

I can't really "fix" anything here, but if you have a lot of custom modes you may want to do a backup of a config file :)

Good luck

**Comment by faslanetech** [comment]
yeah I'm not sure either. Just thought I'd post just in case it wasn't an isolated case. Could have been a Win11 update r something. no idea but it's all good if course  I only have one extra custom mode now where I've set it to ramps up the fans fast and basically stay on almost full blast so I can use it in demanding games with settings cranked up that will make it run very hot  so I made one with full blast fans almost for these specific cases and thank you for the tip on the config backup. I'm going to make one now in fact. again, great little app and awesome support thank you

Fas
